### ISSUE TITLE
Runs log the Menas API run URL twice

### ISSUE THREAD

**Issue by GeorgiChochov** [AbsaOSS/enceladus#1003]
## Describe the bug
When a `Standardization` or `Conformance` job finishes it logs the `Menas` API and UI URLs for the specific run. However, the UI URL contains the same URL as the API URL. 

## To Reproduce
Steps to reproduce the behavior OR commands run:
1. Run any `Standardization` or `Conformance` job
2. See the wrong `Menas UI Run URL` get logged

## Expected behaviour
The `Menas` UI URL should log the correct link.

## Screenshots
```
19/10/25 06:07:01 INFO StandardizationJob$: Menas API Run URL: http://localhost:8080/menas/api/runs/test_dataset/2/6
19/10/25 06:07:01 INFO StandardizationJob$: Menas UI Run URL: http://localhost:8080/menas/api/runs/test_dataset/2/246
```

**Comment by GeorgiChochov** [comment]
Fixed by #1004

**Comment by GeorgiChochov** [comment]
## For release notes:
The `Menas` `API` and `UI` Run URLs will now be logged correctly for each `Menas` instance on the environment.
